components/src/components/textfield/textfield.tsx

```diff
@@ -50,20 +50,62 @@ export class Textfield {
 
   /** Change event for the textfield */
   @Event({
+    eventName: 'sddsChange',
     composed: true,
     bubbles: true,
-    cancelable: true,
+    cancelable: false,
   })
-  customChange: EventEmitter;
+  sddsChange: EventEmitter;
+
+  handleChange(event): void {
+    this.sddsChange.emit(event);
+  }
+
+  /** Input event for the textfield */
+  @Event({
+    eventName: 'sddsInput',
+    composed: true,
+    bubbles: true,
+    cancelable: false,
+  })
+  sddsInput: EventEmitter<InputEvent>;
 
   // Data input event in value prop
-  handleInput(e): void {
-    this.value = e.target.value;
+  handleInput(event): void {
```

Comment:
Just a question, but don't we want to update the value before emitting? To mitigate that the value could be out of sync? Just thinking out loud.
```suggestion
  this.value = event.target.value;
  this.sddsInput.emit(event);
```

Reply:
Absolutely!